tests/integration/suite/webhooks-configs.go

```diff
@@ -0,0 +1,344 @@
+package suite
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"time"
+
+	"github.com/formancehq/formance-sdk-go"
+	. "github.com/formancehq/stack/tests/integration/internal"
+	webhooks "github.com/formancehq/webhooks/pkg"
+	"github.com/formancehq/webhooks/pkg/security"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Given("empty environment for webhooks configs", func() {
+	When("inserting 0 config", func() {
```

Comment:
Useless block.

Reply:
It's a reminder for context, I don't think this is useless